subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv_sync.c

```diff
@@ -352,8 +338,7 @@ static void isr_tx(void *param)
 	}
 
 	radio_gpio_lna_setup();
-	radio_gpio_pa_lna_enable(radio_tmr_tifs_base_get() +
-				 ADV_SYNC_PDU_B2B_AFS - 4 -
+	radio_gpio_pa_lna_enable(radio_tmr_tifs_base_get() + ADV_SYNC_PDU_B2B_AFS - 4 + cte_len_us -
 				 radio_tx_chain_delay_get(lll->phy_s, 0) -
```

Comment:
This is 2 times `2 us` JITTER related with active clock accuracy. It is `4 us` because TX clock may be `-2 us` and receiver clock may be `+2 us`.

Reply:
I think @cvinayak has changed a few in the controller in one of his PRs already based on a similar question from me. 

Since it was there before your commit, I won't let it block my approval, but it is probably a good idea to, at one point, change all these occurrences with a macro.

Reply:
In my humble opinion it should be changed. I'll create a separate PR that adds such change through out the controller code.